infra/build/functions/build_project.py

```diff
@@ -195,12 +195,18 @@ def get_env(fuzzing_language, build):
       'HOME': '/root',
       'OUT': build.out,
   }
+
   return list(sorted([f'{key}={value}' for key, value in env_dict.items()]))
 
 
 def get_compile_step(project, build, env, parallel):
   """Returns the GCB step for compiling |projects| fuzzers using |env|. The type
   of build is specified by |build|."""
+  set_git_repo_env = ''  #do nothing
+  if build.sanitizer == 'instrumentor':
```

Comment:
I don't like that the sanitizer here is called instrumentor. That's a pretty ambiguous name given that sanitizers are instrumentation.
Why don't we call it introspector?